apps/nextjs/src/app/aila/help/index.tsx

```diff
@@ -30,196 +44,247 @@ export const HelpContent = () => {
+                  for pupils to complete for homework, suggestions for
+                  alternative art or equipment you can use for your lesson.
+                </OakP>
+              </OakBox>
+              <OakBox ref={creatingRef}>
+                <OakHeading $mb={"space-between-xs"} $font="heading-5" tag="h2">
                   Co-creating your lesson
-                </button>
-              </li>
-              <li>
-                <button
-                  className="mb-14 text-blue underline"
-                  onClick={() => scrollToRefWithOffset(reviewingRef)}
-                >
+                </OakHeading>
+                <OakP $mb={"space-between-xs"}>
+                  Aila, Oak&apos;s AI lesson assistant will guide you through
+                  the process of co-creating each section of your lesson on the
+                  left-hand side of the screen. The lesson output will be
+                  generated on the right-hand side.
+                </OakP>
+              </OakBox>
+              <OakBox ref={reviewingRef}>
+                <OakHeading $mb={"space-between-xs"} $font="heading-5" tag="h2">
                   Reviewing and editing
-                </button>
-              </li>
-              <li>
-                <button
-                  className="mb-14 text-blue underline"
-                  onClick={() => scrollToRefWithOffset(downloadingRef)}
-                >
+                </OakHeading>
+                <OakP $mb={"space-between-xs"}>
+                  Aila will pause at key points during the lesson creation and
+                  will ask you to review the content generated. You can ask Aila
+                  to adapt each section, for example:
+                </OakP>
+                <OakUL className="mb-6 list-disc pl-12">
+                  <OakLI>
+                    increase or decrease the literacy level of your keywords
+                  </OakLI>
+                  <OakLI>
+                    add specific case studies or examples to your explanations
+                  </OakLI>
+                  <OakLI>change the practice tasks for you</OakLI>
+                  <OakLI>
+                    add a narrative for each learning cycle to the Additional
+                    Materials
+                  </OakLI>
+                </OakUL>
+              </OakBox>
+              <OakBox ref={downloadingRef}>
+                <OakHeading $mb={"space-between-xs"} $font="heading-5" tag="h2">
                   Downloading and sharing
-                </button>
-              </li>
-              <li>
-                <button
-                  className="mb-14 text-blue underline"
-                  onClick={() => scrollToRefWithOffset(aiRef)}
-                >
+                </OakHeading>
+                <OakP $mb={"space-between-xs"}>
+                  Once you’ve completed all the sections in the lesson, you’ll
```

Comment:
Should this be &apos; instead of a '?

Reply:
I think it's a `’` rather than `'`, which is correct (and would be an &apos; symbol in raw html)